namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-estate-dev/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: estate-dev
```

Comment:
Can you please change it to prison-estate-dev, same as your folder name.
OR if you like to keep this name, can you please change your folder name.

Reply:
Apologies - over zealous search and replace went on.  Dev prison estates will be on the domain https://estate-dev.prison.service.justice.gov.uk so my replace for the domain affected all the namespaces by accident.  Now fixed :crossed_fingers: